cfpb/cfgov-refresh
# Appops 830

cfgov-refresh's docker-stack.yml currently runs the refresh-data.sh every time. This is unnecessary considering the postgres service does not redeploy when just app code changes.

This causes a couple issues:

Data modifications are lost between deploys
It makes the deploys slow.  It is an expensive operation to refresh the database, pulling down the big DB archive and applying it to the running container. 

## Additions

- docker-stack-refresh-db.yml
- REFRESH_DB parameter to Jenkinsfile
- Refresh DB step to Jenkinsfile
- Conditionals to refresh db and deploy steps

## Removals

-

## Changes

-

## Testing

1. Run an adhoc deploy with refresh db = true
2. Check for django wagtail table in postgres (wagtailcore_page suggested) 
3. Run an adhoc deploy with refresh db = false
4. Check for django wagtail table in postgres (wagtailcore_page suggested) 

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
